test/spec/index.spec.js

```diff
@@ -14,33 +16,147 @@ describe('index.js', function() {
       var parsedOptions = krabby._getOptions(options);
       assert(_.isEqual(options, parsedOptions));
     });
+    it('finds a krabby config when found inside of another object', function() {
```

Comment:
I don't understand the it statement for this one? Are you just testing that it doesn't throw away opts?